saleor/payment/gateways/stripe/plugin.py

```diff
@@ -177,13 +178,19 @@ def process_payment(
         payment_method_id = data.get("payment_method_id") if data else None
 
         setup_future_usage = None
+        # DEPRECATED
         if payment_information.reuse_source:
             setup_future_usage = data.get("setup_future_usage") if data else None
 
         off_session = data.get("off_session") if data else None
 
         payment_method_types = data.get("payment_method_types") if data else None
 
+        if not setup_future_usage:
+            store = payment_information.store_payment_method
+            if store != StorePaymentMethodEnum.NONE:
+                setup_future_usage = store.lower()
```

Comment:
shouldn't be `store.value` ?

Reply:
Also, I think the better way would be to map our enums to Stripe's values. Even if they're currently the same, it should be safer.